src/pages/home/report/ReportActionsView.js

```diff
@@ -281,6 +333,9 @@ ReportActionsView.propTypes = propTypes;
 ReportActionsView.defaultProps = defaultProps;
 
 export default withOnyx({
+    reports: {
```

Comment:
I think by binding the `ReportActionsView` to the `reports` key in Onyx, you would cause it to re-render whenever _any_ report changes, even though this is only rendering `reportActions` from a specific report. So I think we might want to connect it to Onyx like this:

```javascript
export default compose(
    withOnyx({
        currentlyViewedReportID: {
            key: ONYXKEYS.CURRENTLY_VIEWED_REPORTID,
        },
    }),
    withOnyx({
        report: {
            key: ({currentlyViewedReportID}) => `${ONYXKEYS.COLLECTION.REPORT}${currentlyViewedReportID}`,
        },
        reportActions: {
            key: ({reportID}) => `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`,
            canEvict: props => !props.isActiveReport,
        },
        session: {
            key: ONYXKEYS.SESSION,
        },
    }),
)(ReportActionsView);
```

Reply:
That is a very good point.

Reply:
I'm having some problems with this, Pushed a new commit and added a new comment below explaining it.

Reply:
Done!